__init__.py

```diff
@@ -228,7 +193,7 @@ def ctrl_chg_day(bs):
     if bs & BUTTON_SEL_LONG:
         MODE = DISPLAY
     elif bs & BUTTON_SEL:
-        MODE = CHANGE_HOURS
+        MODE = CHANGE_THEME
```

Comment:
I initially had the idea of having short presses of `SEL` cycle through clock features / modes
(e.g. stopwatch, timer, alarm, etc...) and maybe having a settings panel as the last of these modes.

Right now there are no other features like this, so this is probably fine, but I just wanted to check in to tell you what I had in mind.

I pretty much based the control sheme on the G-Shock digital watches (see below), with long-pressing MODE being a stand-in for the ADJUST button.

![g-shock](https://i.imgur.com/XmCvmna.jpg)